### PR TITLE
Feature mappage

### PR DESCRIPTION
**Pull request by taehioum**
I have created a Map Component.

It uses google maps API to search nearby locations.

On submit, it sends chosen location's name, coordinates(latitude, longitude) to the redux store.

Other functionalities like redirection, and error handling has not been implemented yet.

### CODE REVIEW

**Comment by Minji1234** [comment]
You did really good job! Thanks you for your work. 👍

File: oga/src/components/MapSearchBox/MapSearchBox.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ */
+ 
+/** jshint {inline configuration here} */
+import React, { Component } from 'react';
+import './MapSearchBox.css';
+
+class MapSearchBox extends Component {
+  constructor(props) {
+    super(props);
+    this.clearSearchBox = this.clearSearchBox.bind(this);
```
Comment (Minji1234) [inline oga/src/components/MapSearchBox/MapSearchBox.js]: How about changing this.clearSearchBox.bind(this) to arrow function like this.clearSearchBox => ();?

Comment (taehioum) [reply oga/src/components/MapSearchBox/MapSearchBox.js]: I will look in to that.

**Comment by taehioum** [comment]
Used arrow functions instead of bind(this)

**Pull request merged**